custom_components/sat/climate.py

```
@@ -631,36 +680,37 @@ async def _async_inside_sensor_changed(self, event: Event[EventStateChangedData]
         if new_state is None or new_state.state in (STATE_UNAVAILABLE, STATE_UNKNOWN):
             return
 
-        _LOGGER.debug("Inside Sensor Changed.")
+        _LOGGER.debug(f"Inside sensor changed (%.2f°C).", float(new_state.state))
```

Ruff (0.14.5)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)